### ISSUE TITLE
[other]: confusing logic in Dockerfile

### ISSUE THREAD

**Issue by yf13** [kendryte/k230_sdk#15]
### Other

The recent `Dockerfile` adds a FLAG file which is set to "True" when download from `ai.b-bug.org` fails. The flag is then referenced multiple times later.

When building the docker image from China, the flag is `True` as the download fails always. However, this will further block using pip mirrors at line #51, this is bad as  we need the mirrors.

Is the flag to depict Canaan intranet environment? if so then the logic at line#51 should be reviewed.

**Comment by wuwentao** [comment]
yes,  you are right, we forgot to remove it, thank you very much for your comment.
we will enable this pip mirror config by default in next release.

in addition, we also use github action to release this docker image in `ghcr.io/kendryte/k230_sdk` 
https://github.com/kendryte/k230_sdk/pkgs/container/k230_sdk

**Comment by yf13** [comment]
@wuwentao  this issue seems still exist in latest Dockerfile, at least with v1.1.

**Comment by wuwentao** [comment]
I'm sorry for my mistaken....
when I commit this fix to dev branch, I forgot to commit it to v1.1 release branch, and caused current github v1.1 release not include it....
v1.2 release is under testing, so this changes will be publish in v1.2 release, sorry for the delay.
@yf13

**Comment by yf13** [comment]
That is fine, just sent a PR.
